indico/modules/events/registration/fields/base.py

```diff
@@ -45,6 +45,8 @@ class RegistrationFormFieldBase:
     is_invalid_field = False
     #: whether this field must not be "empty" (falsy value) if required
     not_empty_if_required = True
+    #: whether this field is associated with a file and should be represented as a picture
+    is_picture_field = False
```

Comment:
Based on a quick glance at the current usage of `is_file_field` you can use that one instead and do not need to create a new attribute. That way you'll also automatically get the proper behavior e.g. when purging a field due to the retention period having been reached.

Reply:
Since the picture field is inheriting from the FileField it also has its `is_file_field` set to True. But sure I can remove it since we can just check the fields input_type to know it is a picture.

Reply:
did you even use it somewhere? at least i didn't find it when looking at the PR diff